static_devel/bcxLoader.user.js

```diff
@@ -20,7 +20,7 @@ setTimeout(
 			const n = document.createElement("script");
 			n.setAttribute("language", "JavaScript");
 			n.setAttribute("crossorigin", "anonymous");
-			n.setAttribute("src", "https://jomshir98.github.io/bondage-club-extended/devel/bcx.js?_=" + Date.now());
+			n.setAttribute("src", "https://bananarama92.github.io/bondage-club-extended/devel/bcx.js?_=" + Date.now());
```

Comment:
This should be dropped before merge.
Suggestion for next time: You can edit the script directly in your TM instead of having to commit it as temporary commit. Even better might be making a separate, differently-named script for yourself. To me it feels cleaner, but it is a personal preference in the end.